test/TestHelper.cpp

```diff
@@ -43,6 +43,11 @@ Options::Options()
 			ipcPath = suite.argv[i + 1];
 			i++;
 		}
+		else if (string(suite.argv[i]) == "--testpath" && i + 1 < suite.argc)
```

Comment:
Please also modify the CI scripts accordingly.

Reply:
tests.sh and isolate_tests.py are adjusted

Reply:
I'm not sure if `testpath` is the best way to name this option, but I also cannot think of a better one :)

Reply:
``testroot``maybe?

Reply:
It's not really about the tests, it is rather about the test data.

Reply:
True.

``--testdata``, ``--testdataroot`` or ``--testdatapath`` then :-)? (or hyphenated as ``--test-data-root``, etc.)

At the moment the argument expected for --testpath is the root test directory - libsolidity/syntaxTests is appended to that. One could pass create a different directory tree under test/, e.g. test/data/, and put all test data files in there - then ``--test-data-path`` or an alternative would probably make more sense.

Reply:
Hm, perhaps it is fine as it is :)